### DISCUSSION TITLE
Error creating custom dataset

### DISCUSSION THREAD

**Discussion by Dana136484** [open-mmlab/mmdetection3d#2944] [Q&A]
Following the documentation(https://mmdetection3d.readthedocs.io/en/latest/advanced_guides/customize_dataset.html) I am running in my custom raw dataset:
python3 tools/create_data.py custom --root-path /mnt/Data1/dataset_carla/training --out-dir /mnt/Data1/dataset_carla/training/ --extra-tag custom
I am getting this error:
File "tools/create_data.py", line 356, in <module> raise NotImplementedError(f'Don\'t support {args.dataset} dataset.') NotImplementedError: Don't support custom dataset.
![屏幕截图 2024-04-03 083708](https://github.com/open-mmlab/mmdetection3d/assets/126674286/a5675032-e224-421f-94eb-6898b8ff875f)
I have read the create_data.py, it seems to not support the argument of 'custom'
How to solve it? Thanks!

**Comment by Vish19-code** [comment]
hello, you have to add the data in creat_data.py accroding you data set so this is solved
